oqs-test/README.md

```diff
@@ -7,12 +7,12 @@ OQS-OpenSSH Integration Testing
 
 This directory contains scripts for testing the OQS fork of OpenSSH with liboqs, using all supported algorithms. The [README.md file for the OQS-OpenSSH fork](https://github.com/open-quantum-safe/openssh-portable/blob/OQS-master/README.md) describes the various key exchange and authentication mechanisms supported.
 
-First make sure you have **installed the dependencies** for the target OS as indicated in the [top-level testing README](https://github.com/open-quantum-safe/openssh-portable/blob/OQS-master/README.md).
+First make sure you have **installed the dependencies** for the target OS as indicated in the [top-level README](https://github.com/open-quantum-safe/openssh-portable/blob/OQS-master/README.md).
 
-Testing on Linux and macOS
---------------------------
+Testing on Linux
+-----------------
 
-The scripts have been tested on macOS 10.14, Debian 10 (Buster), and Ubuntu 18.04 (Bionic).
+The tests are guaranteed to run on Ubuntu 18.04 (Bionic).
```

Comment:
"guaranteed to run" is pretty strong language.

Also, do we no longer support running on macOS?

Reply:
We don't run CI on macOS, which is why I removed that text.